.npmrc

```diff
@@ -1 +1,2 @@
-save-exact=true
\ No newline at end of file
+save-exact=true
+@jsr:registry=https://npm.jsr.io
```

Comment:
Why we need this?

Reply:
This just adds a link to the jsr.io package registry, which is a modern version of the npm registry. Paseri is hosted there only, not on npm. https://jsr.io/#why-jsr